src/main/java/uk/gov/companieshouse/confirmationstatementapi/configuration/InterceptorConfig.java

```diff
@@ -146,7 +147,8 @@ private void addCompanyAuthenticationEndpointsInterceptor(InterceptorRegistry re
      */
     private void addTransactionIdValidationInterceptor(InterceptorRegistry registry) {
         registry.addInterceptor(transactionIdValidationInterceptor)
-                .addPathPatterns(TRANSACTIONS);
+                .addPathPatterns(TRANSACTIONS)
+                .excludePathPatterns(FILINGS);
```

Comment:
Is this and following excludePaths required or redundant? (The company interceptor does not has an exclude rule for FILINGS).  I suppose it works but tricky to follow

Reply:
This was added to allow the use of more general patterns for endpoint matching, while ensuring that all interceptors applied to the existing endpoints (specifically filings) remain unchanged.  This approach was preferred over the alternative of using a new specific pattern to add the email endpoint where required, as it will now automatically apply relevant interceptors to any future private endpoints.  The company one does not need an exclude rule because the COMPANY_NUMBER pattern does not match the filings endpoint.

Reply:
So presumably this means that if you ran your new test InterceptorConfigRouteMatchingTest against the original InterceptorConfig (on the parent branch) it would pass.

Reply:
Yes, apart from the intended changes relating to the email endpoint (switching auth type + adding company number validation) - this was how I developed the test cases, & I have just confirmed it still produces expected results.